src/midi.js

```diff
@@ -34,7 +36,16 @@ const midi = (notes, fileName) => {
 		}
 	});
 
-	fs.writeFileSync(fileName, file.toBytes(), 'binary');
+	writeFile("./output/" + fileName, file.toBytes(), 'binary');
 }
 
+function writeFile(path, contents, cb) {
+  mkdirp(getDirName(path), function (err) {
+    if (err) return cb(err);
+
+    fs.writeFileSync(path, contents, cb);
+  });
+}
+
+
```

Comment:
The issue for the test was when it runs the first time it creates the folder Asynchronously.
if you update your writeFile method to: 
```

	function writeFile(outPath, contents, options) => {
	  let outputPathCreated = mkdirp.sync(getDirName(outPath));
	  if (outputPathCreated) {
	  	fs.writeFileSync(outPath, contents, options);
	  } else { console.log('Error: Unable to create output directory'); }
        }

```

Reply:
@ChrisAntaki Cheers for this, I'll implement it now! Its been a bit of a confusing error for me to work out. I appreciate everyones help 👍